src/input_utils.py

```diff
@@ -13,33 +13,52 @@
 # limitations under the License.
 
 import attr
```

Comment:
Nit: Imports should be grouped into a standard library block first (e.g. with `typing`) followed by a third party block (where `attr` belongs).  More style guide info here:
https://google.github.io/styleguide/pyguide.html#313-imports-formatting